src/core/openthread-instance.h

```diff
@@ -100,6 +100,9 @@ typedef struct otInstance
     otLogLevel mLogLevel;
 #endif // OPENTHREAD_CONFIG_ENABLE_DYNAMIC_LOG_LEVEL
 
+    ot::Timer mTimer;
+    static void HandleInstanceTimer(void *aContext);
```

Comment:
Because `Timer` class does not have one API to register the callback function pointer now(need to pass in the handler parameter when initializing one timer instance), we need one callback on `otInstance` class to accept the registered application timer handler and trigger this handler when timer on `otInstance` expires. So here I would like to define such one callback within `otInstance` class. Please correct me if my understanding is wrong.